tests/code_wars/test_highest_scoring_word.py

```diff
@@ -0,0 +1,22 @@
+from code_challenges.code_wars.highest_scoring_word import high
+
+
+def test_high():
+    actual = high("it will take a string")
+    expected = "string"
+
+    assert actual == expected, f"Returned {actual} instead of {expected}"
+
+
+def test_high_nultiple_same_score():
+    actual = high("ade j ee gc")
+    expected = "ade j ee gc"
+
+    assert actual == expected, f"Returned {actual} instead of {expected}"
+
+
+def test_high():
```

Comment:
This function is named the same as the top one. This isn't allowed. Are you sure both are running?